scripts/RAG.py

```diff
@@ -48,4 +49,26 @@ def generate_answer(self, query_text, k=4):
         except Exception as e:
             logging.error(f"Error generating answer: {e}")
             return "Error generating answer."
+    
+    def stream_answer(self, query_text, k=4):
```

Comment:
in other function signature you are adding type-hints, but not here, so you could add it.